Makefile

```diff
@@ -1,22 +1,34 @@
 export GO111MODULE=on
 
+#######################
+### Current Build Properties
+VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
+COMMIT := $(shell git log -1 --format='%H')
+BUILD_TAGS = netgo
+
+
+BUILD_FLAGS = -tags "$(BUILD_TAGS)" -ldflags \
```

Comment:
so this is to specify the specific sdk version you want to build with?

Reply:
no, it's not. Go allows you do set package variables on build.

https://github.com/cosmos/cosmos-sdk/blob/master/version/version.go#L25, if you notice, there is nothing set.

on `make install`, Version/commit-hash package variables will be built into the binary (-ldflag). This is why i suggested having plasmacli make use of the same version command. To update the version, we simply just have to add a new tag a commit.

The netgo simply allows the binaries to be more portable (pure go networking stack)

Reply:
In the next develop -> master PR, it should include a `1.0.0` tag.

Reply:
I see, nice addition!

Reply:
what do you think about having plasmacli use the same version command? Such that the commit hash and version will be consistent on plasmad/plasmacli

Reply:
Makes sense to me

Reply:
Great, I'll update this. Going to put up one more PR to just include more test coverage and godoc and we're ready for 1.0.0